can-stache.js

```diff
@@ -39,6 +39,7 @@ function stache(template){
 
 	// Remove line breaks according to mustache's specs.
 	if(typeof template === "string") {
+		template = mustacheCore.cleanWhitespaceControl(template);
```

Comment:
I'm not thinking hard right now, but it would be good to make sure this is going to work if we get the parsed HTML `AST` from something like `steal-stache`.  Is it possible to do this via changing `parseData` similar to how __slots__ and __recursive templates__ are working?  Maybe not.  And that might be a far less easy to understand solution.